src/AwesomizeExpress.re

```diff
@@ -0,0 +1,39 @@
+/* @TODO - Awesomizer needs to support a context and this integration needs
+         to accept a function which can generate that context from the request
+         object.
+   */
+let make = (~schema, ~decoder, ~encoder, ~handler) => {
+  let beltMapToJson = map =>
```

Comment:
All the internal functions should be moved out of the `make` function. Because an interface file is provided they no longer need to be inner functions to _hide_ them.

Reply:
Is there anyway to make said helper functions private in the ReasonML ecosystem?  Or is that a concern that we need not worry about?

Reply:
```
If React.rei isn't provided, the signature of React.re defaults to exposing all the fields of the module.
```

Seems like if I don't include the signatures of the internal functions in the interface file, the internal functions will be exposed.  Awesome!